routes/users.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+
+// Initaialize router
+const router = express.Router();
+
+// Create an array to store user data
+let users = [];
+
+// Find all users
+router.get('/users', (req, res) => {
+    res.json(users)
+})
+
+// Create users
+router.post('/create', (req, res) => {
```

Comment:
why no validation for the data when creating an account?

Reply:
and i also i will also advice that you abstract this to controllers and then services. instead of just doing all this on the routes. you can research on how to do this

Reply:
Just had a conversation with Mr Ope, he has sent me links on how I will implement the validation, will work on that